ddtrace/contrib/requests/patch.py

```diff
@@ -39,8 +40,9 @@ def _traced_request_func(func, instance, args, kwargs):
 
     method = kwargs.get('method') or args[0]
     url = kwargs.get('url') or args[1]
+    parsed_url = urlparse(url)
 
-    with tracer.trace("requests.request", span_type=http.TYPE) as span:
+    with tracer.trace("requests.request", service=parsed_url.netloc, span_type=http.TYPE) as span:
```

Comment:
@awang is this one a safe choice? since we're doing something similar for Ruby, I want to be sure this is what users are expecting. Changing the `resource` is too risky because of high cardinality, but here the base location must be limited "in most of the cases". What do you think? What about dynamic address resolutions such as the AWS public DNS that could have a high cardinality?

Reply:
@palazzem how high of a cardinality would the AWS public DNS case be?  i assumed it would be on the order of 100 unique DNS entries or less, in which case i felt it would be fine

Reply:
@awang yes that would be the cardinality more or less; in that case we can proceed with that, keeping the `resource` fixed as we're doing now. Will work to have this PR merged too, thanks!